From: michael <michael@b3059339-0415-0410-9bf9-f77b7e298cf2>
Date: Tue, 5 May 2009 01:34:16 +0000 (+0000)
Subject: Change VOFW for x86 to 5120, it allows larger images to be scaled and was
X-Git-Url: http://git.ffmpeg.org/?p=libswscale;a=commitdiff_plain;h=c9d08ac2f27b3c7a18db9d2ef417673f77be405f

Change VOFW for x86 to 5120, it allows larger images to be scaled and was
not slower. Other archs are not changed as the larger VOFW was slower on PPC.


git-svn-id: file:///var/local/repositories/mplayer/trunk/libswscale@29256 b3059339-0415-0410-9bf9-f77b7e298cf2
---

diff --git a/swscale_internal.h b/swscale_internal.h
index 3995887..eb11166 100644
--- a/libswscale/swscale_internal.h
+++ b/libswscale/swscale_internal.h
@@ -33,7 +33,12 @@
 
 #define MAX_FILTER_SIZE 256
 
-#define VOFW 2048
+#if ARCH_X86
+#define VOFW 5120
+#else
+#define VOFW 2048 // faster on PPC and not tested on others
+#endif
+
 #define VOF  (VOFW*2)
 
 #ifdef WORDS_BIGENDIAN
